src/main.cpp

```diff
@@ -1161,27 +1161,40 @@ int main(int argc, char**argv)
 
   DiameterResolver* diameter_resolver = NULL;
   RealmManager* realm_manager = NULL;
+  Alarm* hss_peer_connection_alarm =NULL;
 
   if (hss_configured)
   {
     diameter_resolver = new DiameterResolver(dns_resolver,
                                              af,
                                              options.diameter_blacklist_duration);
+
+    hss_peer_connection_alarm = new Alarm(alarm_manager,
+                                          "homestead",
+                                          AlarmDef::HOMESTEAD_HSS_REALM_PEER_ERROR,
+                                          AlarmDef::MINOR);
+
     if (options.force_hss_peer.empty())
     {
       realm_manager = new RealmManager(diameter_stack,
                                        options.dest_realm,
                                        options.dest_host,
                                        options.max_peers,
-                                       diameter_resolver);
+                                       diameter_resolver,
+                                       hss_peer_connection_alarm,
+                                       "Homestead",
```

Comment:
How about passing in CLEARED/ERROR ENT logs, rather than "Homestead"/"HSS"? I don't think commonising the ENT log gives you anything (and it's something we're moving away from in the CM).